website/docs/r/subscription_cost_management_export.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Cost Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_subscription_cost_management_export"
+description: |-
+  Manages an Azure Cost Management Export for a Subscription.
+---
+
+# azurerm_subscription_cost_management_export
+
+Manages an Azure Cost Management Export for a Subscription.
+
+## Example Usage
+
+```hcl
+data "azurerm_subscription" "example" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                = "example-storage-account"
+  resource_group_name = azurerm_resource_group.example.name
+
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_resource_group_cost_management_export" "example" {
+  name                    = "example"
+  subscription_id         = azurerm_subscription.example.id
+  recurrence_type         = "Monthly"
+  recurrence_period_start = "2020-08-18T00:00:00Z"
+  recurrence_period_end   = "2020-09-18T00:00:00Z"
+
+  export_data_storage_location {
+    storage_account_id = azurerm_storage_account.example.id
+    container_name     = "examplecontainer"
```

Comment:
presumably we should create a container here too?

Reply:
Azure creates the container if it doesn't exist, but I agree that we should explicitly create it with Terraform to avoid the creation of unmanaged resources.